external/troubleshooting/sony-vaio-tinkering.md

```diff
@@ -11,36 +11,37 @@ redirect_from:
 Instructions for getting your Sony Vaio Z laptop working with Qubes/Linux
 =========================================================================
 
-Sony Vaio Z are great laptops -- they are very powerful, yet compact. The newer models, starting from Z12 are, however, not very well supported by Linux kernels (at least 2.6.34 that we currently use in Dom0) and thus some tinkering is needed to get Qubes working on those machines.
+The following issues were reported on Qubes 3.2 and may not be prevalent on Qubes 4.0. 
```

Comment:
I think it might be even older than that. I'm not sure if anyone even uses Qubes on a Sony Vaio anymore.